docs/guide.md

```diff
@@ -2593,12 +2641,24 @@ There are two ways to create an immutable **List**, **Set**, **Bag**, **Stack**
 <!-- -->
 
 ```java
+
+// 1. Calling toImmutable()
+ImmutableList<String> list = 
+    Lists.mutable.with("a", "b", "c").toImmutable(); // creates a list of elements a,b,c
+ImmutableSet<String> set0 =  
+    Lists.mutable.with("a", "b", "a", "c").toSet().toImmutable(); // creates a set of elements a,b,c
+ImmutableSet<String> set1 =  
+    Sets.mutable.with("a", "b", "a", "c").toImmutable(); // creates a set of elements a,b,c
+ImmutableMap<Integer, String> map = 
+    Maps.mutable.with(1, "a", 2, "b", 3, "c").toImmutable(); // creates a map with keys 1,2,3
+
+//2. Using factory classes
 ImmutableList<String> emptyList_i = 
     Lists.immutable.empty();  // creates an empty list
-ImmutableList<String> list_b = 
-    Lists.immutable.with("a", "b", "c"); // creates a list of elements a,b,c
-ImmutableList<String> list_c = 
-    Lists.immutable.of("a", "b", "c");  // creates a list of elements a,b,c
+ImmutableSet<String> set =  
```

Comment:
That's not quite what I was expecting. Let me write it down what I was expecting. All these 3 should be present for clarity.

```
ImmutableList<String> emptyList_i = 
    Lists.immutable.empty();  // creates an empty list
ImmutableList<String> list_b = 
    Lists.immutable.with("a", "b", "c"); // creates a list of elements a,b,c
ImmutableList<String> list_c = 
    Lists.immutable.of("a", "b", "c");  // creates a list of elements a,b,c

ImmutableSet<String> emptySet_i = 
    Sets.immutable.empty();  // creates an empty Set
ImmutableSet<String> Set_b = 
    Sets.immutable.with("a", "b", "c"); // creates a Set of elements a,b,c
ImmutableSet<String> Set_c = 
    Sets.immutable.of("a", "b", "c");  // creates a Set of elements a,b,c

ImmutableMap<Integer, String> map = 
    Maps.immutable.empty(); // creates an empty map
ImmutableMap<Integer, String> map = 
    Maps.immutable.with(1, "a", 2, "b", 3, "c"); // creates a map with keys 1,2,3

```